src/postprocess/band_structure.jl

```diff
@@ -23,10 +23,11 @@ function high_symmetry_kpath(basis; kline_density=20)
 end
 
 function compute_bands(ham::Hamiltonian, kpoints, n_bands;
-                       eigensolver=lobpcg_hyper, tol=1e-5)
+                       eigensolver=lobpcg_hyper, tol=1e-5, show_progress=true)
```

Comment:
kwargs and pass to diagonalise_all_kblocks ?

Reply:
Why not ...

Reply:
I meant use it for tol and show_progress

Reply:
Does not work, because it has different defaults.